### ISSUE TITLE
Table header in Markdown Web preview is black

### ISSUE THREAD

**Issue by ghost** [subethaedit/SubEthaEdit#175]
## Short Description
Table header in Markdown Web preview is black. I tried to find an option in preferences, but could not find any, except for the code syntax coloring.
## Steps to Reproduce the Issue
```markdown
## Statistics
|Language|files|blank|comment|code|description
|:-------|-------:|-------:|-------:|-------:|:-------
|**HTML**|14|1|13|327|web pages
|**SVG**|1|0|0|73|resources
|---------|---|---|---|---|
|**SUM**:|15|1|13|400|
```
Command-R shows the web preview on the left.
## Expected Result
![black-headers-in-subethaedit-expected](https://user-images.githubusercontent.com/66477240/83880806-057c5000-a740-11ea-907e-73f82673e18b.png)
## Actual Result
![black-headers-in-subethaedit](https://user-images.githubusercontent.com/66477240/83880300-39a34100-a73f-11ea-8b9f-ecdcd94bad21.png)

## Environment

- SubEthaEdit: Version 5.1.3 (9672)
- System: Version 10.15.5 (Build 19F101)
- Language: English

```
Hardware:

    Hardware Overview:

      Model Name: MacBook Pro
      Model Identifier: MacBookPro15,1
      Processor Name: 6-Core Intel Core i7
      Processor Speed: 2,6 GHz
      Number of Processors: 1
      Total Number of Cores: 6
      L2 Cache (per Core): 256 KB
      L3 Cache: 9 MB
      Hyper-Threading Technology: Enabled
      Memory: 16 GB
      Boot ROM Version: 1037.120.87.0.0 (iBridge: 17.16.15300.0.0,0)
      Serial Number (system): C02XJLRQJG5J
      Hardware UUID: C88A2339-A8B2-520E-A66C-7C708AA08381


```

**Comment by ghost** [comment]
If it requires work, I am willing to create a pull request, if you could point me to the resources that should be adjusted to solve this.
No guarantees though.

Thanks for looking into this.

**Comment by monkeydom** [comment]
Thanks for reporting. This is already fixed in main line (although unreleased). 
Fixed in a91c3d52b2443ae14d6fd5779307e34611f39aa5

Installing the attached mode should fix it for you.
[Markdown.seemode.zip](https://github.com/subethaedit/SubEthaEdit/files/4810088/Markdown.seemode.zip)
